modules/engine/src/main/scala/com/gsk/kg/engine/data/ToTree.scala

```diff
@@ -55,10 +55,15 @@ object ToTree extends LowPriorityToTreeInstances0 {
           case DAG.Filter(funcs, expr) =>
             Node("Filter", funcs.map(_.toTree).toStream #::: Stream(expr))
           case DAG.Join(l, r) => Node("Join", Stream(l, r))
-          case DAG.OffsetLimit(offset, limit, r) =>
+          case DAG.Offset(offset, r) =>
             Node(
-              "OffsetLimit",
-              Stream(Leaf(offset.toString), Leaf(limit.toString), r)
+              "Offset",
+              Stream(Leaf(offset.toString), r)
```

Comment:
I think you can change this to:

```suggestion
              Stream(offset.toTree, r)
```

If you've instances for `Show[Int]`

Reply:
Cool! It worked!